src/test/TEST_sstruct/maxwell.saved.lassen

```diff
@@ -1,12 +1,12 @@
 # Output file: maxwell.out.0
 Iterations = 10
-Final Relative Residual Norm = 2.480525e-09
+Final Relative Residual Norm = 2.480102e-09
 
```

Comment:
the result changes are expected, since we move more computation of maxwell solvers to GPUs.